src/background/background-backup-wallet-salt-handler.ts

```diff
@@ -0,0 +1,88 @@
+import { logger } from '@shared/logger';
+import { gaiaUrl } from '@shared/constants';
+import { createWalletGaiaConfig, generateWallet } from '@stacks/wallet-sdk';
+import { GaiaHubConfig, uploadToGaiaHub } from '@stacks/storage';
+import { decryptContent, encryptContent, getPublicKeyFromPrivate } from '@stacks/encryption';
+import { fetchPrivate } from '@stacks/common';
+
+interface WalletDataMigrationConfig {
+  version: string;
+  oldWalletSalt: string;
```

Comment:
Is there something more descriptive we can use here than `old`? I'm losing track of what all the salt versions are here.

Reply:
Basically this is not the current wallet salt (so can't use `salt`), and it might not always be the previous one (can't use `previousSalt`). I can't use a specific date since it depends on the day the user updates its wallet and we don't know in advance. I can use the current wallet version appended.
like `salt_v3.2.0` wdyt ?

Reply:
Thinking on this, would it be better to be more specific, like `walletSaltThatGeneratesIncorrectAppKey`?